src/routers/datatypeRouters/nodes/variants.ts

```diff
@@ -0,0 +1,113 @@
+import { z } from 'zod'
+import { publicProcedure } from '../../../trpc'
+import { loadSchemaConfig } from '../../genericRouters/genericRouters'
+import { RouterFilterBy } from '../../genericRouters/routerFilterBy'
+import { RouterFilterByID } from '../../genericRouters/routerFilterByID'
+import { preProcessRegionParam, paramsFormatType } from '../_helpers'
+
+const schema = loadSchemaConfig()
+
+const frequencySources = z.enum([
```

Comment:
this is a sample of sources, I will recalculated all possible sources once all data is loaded in the database